Holovibes/includes/API.hh

```diff
@@ -760,6 +760,11 @@ void display_reticle(bool value);
  */
 void reticle_scale(float value);
 
+/*! \brief Set the new value of the registration zone for the circular mask. Range ]0, 1[.
+ *  \param[in] value The new zone value.
+ */
+void registration_zone(float value);
```

Comment:
maybe rename not to set_registration_zone but update_registration_zone ? (indicate this is a setter)